### DISCUSSION TITLE
How to ignore dark mode?

### DISCUSSION THREAD

**Discussion by Gerodot** [mdbootstrap/TW-Elements#1637] [Support from community] [answered]
Hi all. 
I searched a lot of places and tried to solve the problem myself, but could not. The problem is that the table header has Multiselect blocks with a white background. The background is independent of the design theme and always has a white background. How can I prevent the font color and other things from changing when the dark theme is activated?

**Comment by juujisai** [comment, marked as answer]
Hi! Did you try adding `darkMode="class"` to your `tailwind.config.js` file?

```
module.exports = {
...
  darkMode: "class"
};
```
